src/update_sdk.cmd

```diff
@@ -0,0 +1,43 @@
+REM This script replaces the SDK folder with the latest bits from the Azure SDK repository.
+REM It removes some files we do not need.  It currently targets the develop branch until fixes are
+REM merged into master.
+
+if exist "%~dp0\temp" rd /s /q "%~dp0\temp"
+if exist "%~dp0\sdk" rd /s /q "%~dp0\sdk"
+
+mkdir "%~dp0\temp"
+mkdir "%~dp0\sdk"
+
+cd /D "%~dp0\temp"
+
+git clone -b develop https://github.com/Azure/azure-iot-sdks.git --recursive
+
+cd "%~dp0\temp\azure-iot-sdks\"
+
+git checkout develop
+
+git reset --hard HEAD
+
+git pull origin develop
+
```

Comment:
Are the last 3 git commands needed? It looks like the script deletes the old folder and clones, so it should be update to date.